src/vex.ts

```diff
@@ -37,46 +35,52 @@ export const Vex = {
     } else {
       return arr;
     }
-  },
+  }
 
   /** Check if array `arr` contains `obj`. */
   // eslint-disable-next-line
-  Contains: (arr: any[], obj: any) => {
+  static contains(arr: any[], obj: any): boolean {
     let i = arr.length;
     while (i--) {
       if (arr[i] === obj) {
         return true;
       }
     }
     return false;
-  },
+  }
 
   // Get the 2D Canvas context from DOM element `canvas_sel`.
-  getCanvasContext: (canvas_sel: string): RenderingContext => {
-    if (!canvas_sel) {
-      throw new RuntimeError('BadArgument', 'Invalid canvas selector: ' + canvas_sel);
+  static getCanvasContext(canvasSelector: string): RenderingContext {
+    if (!canvasSelector) {
+      throw new RuntimeError('BadArgument', 'Invalid canvas selector: ' + canvasSelector);
     }
 
-    const canvas = document.getElementById(canvas_sel) as HTMLCanvasElement;
+    const canvas = document.getElementById(canvasSelector) as HTMLCanvasElement;
     if (!(canvas && canvas.getContext)) {
       throw new RuntimeError('UnsupportedBrowserError', 'This browser does not support HTML5 Canvas');
     }
 
     return canvas.getContext('2d') as RenderingContext;
-  },
+  }
 
   /** Benchmark. Run function `f` once and report time elapsed shifted by `s` milliseconds. */
   // eslint-disable-next-line
-  BM: (s: any, f: any) => {
+  static benchmark(s: any, f: any): void {
     const start_time = new Date().getTime();
     f();
     const elapsed = new Date().getTime() - start_time;
     log(s, elapsed + 'ms');
-  },
+  }
 
   // Get stack trace.
-  StackTrace: (): string | undefined => {
+  static stackTrace(): string | undefined {
     const err = new Error();
     return err.stack;
-  },
-};
+  }
+
+  // Backwards compatability with 3.0.9.
+  static RERR: RuntimeError;
```

Comment:
There were a few smaller things like this, where I reintroduced an API that had disappeared sometime between 3.0.9 and today. For example, Vex.RERR was used a lot in VexTab.